src/mmap.cpp

```diff
@@ -192,17 +224,23 @@ namespace {
 			;
 	}
 
+#endif
+
 } // anonymous
 
 file_handle::file_handle(string_view name, std::int64_t const size
 	, open_mode_t const mode)
+#ifdef TORRENT_WINRT
+	: m_fd(create_file(name, mode))
+#else
 	: m_fd(CreateFileW(convert_to_native_path_string(name.to_string()).c_str()
```

Comment:
I wonder if it would  be an improvement to move this call to `CreateFileW()` into a `create_file()` function as well, with the same signature as the one for WinRT. That way you could remove the `#ifdef` here, and instead just rely on the `create_file()` having the right definition based on build mode